packages/react-cooked-bread/src/active-toasts-hook.ts

```diff
@@ -36,12 +35,8 @@ export const useActiveToasts = (maxToasts?: number | undefined) => {
         const { type = 'info', ...rest } = options
         const newToast = { content, id, type, ...rest }
         setToasts((prevToasts) => {
-          if (typeof maxToasts === 'number' && maxToasts < prevToasts.length + 1) {
-            if (maxToasts > 0) {
-              return [...prevToasts.slice(1), newToast]
-            } else {
-              return []
-            }
+          if (maxToasts && maxToasts < toasts.length + 1) {
```

Comment:
```suggestion
          if (maxToasts && maxToasts < prevToasts.length + 1) {
```

I believe that should prevent having to include `toasts` in the dependency array (see lint warning below).